lib/services/keyVault/lib/keyVaultClient.d.ts

```diff
@@ -5526,7 +5717,7 @@ declare class KeyVaultClient extends AzureServiceClient {
    *
    * @reject {Error|ServiceError} - The error object.
    */
-  purgeDeletedStorgeAccountWithHttpOperationResponse(vaultBaseUrl: string, storageAccountName: string, options?: { customHeaders? : { [headerName: string]: string; } }): Promise<HttpOperationResponse<void>>;
+  purgeDeletedStorageAccountWithHttpOperationResponse(vaultBaseUrl: string, storageAccountName: string, options?: { customHeaders? : { [headerName: string]: string; } }): Promise<HttpOperationResponse<void>>;
```

Comment:
Is this a breaking change?

Reply:
@RikkiGibson yes this is an intentional breaking change. This API is only in our preview release, so we are fixing the typo before we GA.

Reply:
Sounds good 👍